mis_builder/models/aep.py

```diff
@@ -281,14 +284,20 @@ def do_queries(self, date_from, date_to,
                                         ['debit', 'credit', 'account_id'],
                                         ['account_id'])
             for acc in accs:
+                company = account_model.browse(acc['account_id'][0]).company_id
```

Comment:
Hm. Indeed this float_compare occurs before currency conversion, so you are probably right, yes.